src/main/java/uk/gov/hmcts/reform/blobrouter/reconciliation/controller/ReconciliationController.java

```diff
@@ -58,7 +68,12 @@ public SuccessfulResponse uploadSupplierReport(
     ) {
         validateAuthorization(authHeader);
         UUID uuid = service.saveSupplierStatement(date, supplierStatement);
-        return new SuccessfulResponse(uuid.toString());
+
+        if (dateRelevantForCurrentReconciliationChecker.isTimeRelevant(ZonedDateTime.now(clockProvider.getClock()))) {
+            return new SuccessfulResponse(uuid.toString());
+        } else {
+            return new SuccessfulResponse(uuid.toString(), "IRRELEVANT REPORT");
```

Comment:
Do we actually want to add it to db as well or something? was not in scope, so probably not, but maybe would b helpful?